src/forms/Helpers/helpers.js

```diff
@@ -0,0 +1,6 @@
+export const composeValidators = (...validators) => value =>
+  validators.reduce((error, validator) => error
```

Comment:
You are basically doing some in here, am I right? Since you are always sending the first `error` you encounter it would be better to use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) so when we encounter first error we will show it to user.

Reply:
it would be nice but i actually need the error message and some returns only `true/false`